Framework/DataHandling/src/SaveAscii2.cpp

```diff
@@ -275,63 +275,14 @@ void SaveAscii2::exec() {
   file.close();
 }
 
-/**writes a spectra to the file using an iterator
-@param spectraItr :: a set<int> iterator pointing to a set of workspace IDs to
-be saved
+/** Writes a spectrum to the file using a workspace index
+@param wsIndex :: an integer relating to a workspace index
 @param file :: the file writer object
 */
-void SaveAscii2::writeSpectra(const std::set<int>::const_iterator &spectraItr,
```

Comment:
I think so... Code was not exactly the same, one was:

```
if (A & B)
   // Do something (I)
else if (!B)
   // Do something (II)
else
   // Do something (III)
```

and the other was:

```
if (!B)
  // Do something (II)
else if (A & B)
  // Do something (I)
else
  // Do something (III)
```

but both should do the same.